packages/playground/src/components/debug-menu.ts

```diff
@@ -410,7 +409,9 @@ export class DebugMenu extends LitElement {
 
         <div
           class="edgeless-toolbar"
-          style=${'display:' + (this.mode === 'edgeless' ? 'flex' : 'none')}
+          style=${
+            'display:none' /*'display:' + (this.mode === 'edgeless' ? 'none' : 'none')*/
```

Comment:
remove?

Reply:
Well I'd rather keep it here till the new self-contained toolbar is landed.

/cc @thorseraq

Reply:
Thanks, current debug menu is a good reference